### ISSUE TITLE
fillChromPeaks for XChromatograms 

### ISSUE THREAD

**Issue by stanstrup** [sneumann/xcms#340]
I don't need it ATM but for the processing of XChromatograms objects to be complete fillChromPeaks would be the last item missing.

**Comment by jotsetung** [comment]
Right. We have to implement that. It *should* be fairly simple... I hope.

**Comment by jmorim** [comment]
Was this ever addressed? I have SRM data where I can't continue with analysis because of this.

**Comment by jorainer** [comment]
no, sorry, not implemented yet. busy with other things, but we plan to update/improve the Chromatogram infrastructure next year. so along with that we should also work on that.
